src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+
+  if (days >= 7) {
+    return (days * cost) - 50;
+  }
+
+  if (days >= 3) {
+    return (days * cost) - 20;
+  }
+
+  return days * cost;
```

Comment:
- move all numbers to the variables as it will make a code more readable and cleaner 
- `(days * cost)` is duplicated, so make sense to move it to the variable